libs/client-websocket/src/error.rs

```diff
@@ -59,7 +59,7 @@ pub enum Error {
   #[error("URL error: {0}")]
   Url(#[from] UrlError),
   #[error("HTTP error: {}", .0.status())]
-  Http(Response<Option<Vec<u8>>>),
+  Http(Box<Response<Option<Vec<u8>>>>),
```

Comment:
More recent versions of clippy reminds that with this structure inlined directly, the Error enum struct is over 136B in size.